src/sentry/api/bases/organization_integrations.py

```diff
@@ -35,7 +36,56 @@ def validate_integration_id(integration_id):
             raise Http404
 
     @staticmethod
-    def get_organization_integration(organization, integration_id) -> RpcOrganizationIntegration:
+    def get_organization_integration(
```

Comment:
Having control + region base classes that implement the same interface is a nice design :clap: